bosh/opsfiles/production.yml

```diff
@@ -0,0 +1,75 @@
+- type: replace
+  path: /name?
+  value: shibboleth-production
+
+- type: replace
+  path: /instance_groups/name=idp/properties/idp?
+  value: 
+    login:
+      forgotPasswordLink: https://account.fr.cloud.gov/forgot-password
+    totp:
+      issuerName: "cloud.gov"
+    entityID: https://idp.fr.cloud.gov
+    scope: fr.cloud.gov
+    metadata:
+      providers:
+      - id: fr
+        metadataURL: https://login.fr.cloud.gov/saml/metadata
+    sslPrivateKey: ((idp_cert.private_key))
+    sslCertificate: ((idp_cert.certificate))
+    signing: 
+      cert: ((idp_signing.certificate))
+      key: ((idp_signing.private_key))
+    encryption: 
+      cert: ((idp_encryption.certificate))
+      key: ((idp_encryption.private_key))
+    passwords: 
+      sealer: ((sealer_password))
+
+## Here is an example of stuff you might put in here.
```

Comment:
What is this for?

Reply:
This was what was in the secrets.example.yml file previously, presumably so that you could know what to override.  I just moved it here so that it's data would not be lost.

Reply:
OK, but is it useful to include here? The data won't be lost anyway, because of git.